docs/tool_help/create_breach_locations_help.py

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I'm not sure if docs should be a Python subpackage. It contains only text files and class diagrams. Now it's a mix of python files and other type of files, so now it's not clear to me what the purpose is any more: general documentation or something else?

Also, like Sander said, maybe we shouldn't spread functionality across too many modules.